src/diffusers/video_processor.py

```diff
@@ -30,17 +30,19 @@ def preprocess_video(self, video, height: Optional[int] = None, width: Optional[
         Preprocesses input video(s).
 
         Args:
-            video: The input video. It can be one of the following:
+            video:
```

Comment:
```suggestion
            video (`List[PIL.Image]` or `List[List[PIL.Image]]` or `torch.Tensor` or `numpy.array` or `List[numpy.array]`):
```

this should fix the issue. doc-builder was expecting there to be argument type explicitly written